app-next/app/trips/Trip.module.css

```diff
@@ -1,5 +1,20 @@
-.info h1{
-    color: black;
-    margin-top: 3em;
-  
+
+.info{
+  background-color: #c4c1c1;
+  padding: 20px 0;
```

Comment:
Stick to `em` or `rem` for the padding? 

Otherwise it might looked messed up on mobile devices, I would recommend stciking to one option